Translations/Portugues Brasil.csv

```diff
@@ -4,35 +4,35 @@ Key,Translation,Version,Comment (context for translators)
 <Mouse>,Mouse,,
 abandon,Descartar,,
 abort,Abortar,,
-aborting.expedition.died,Abortando expedição. O tripulante {0} morreu antes de entrar no transporte.,,
-aborting.expedition.target.too.far,Abortando expedição. Estamos muito longe do nosso alvo {0}.,,
+aborting.expedition.died,Abortando Expedição.{0} faleceu.,,
+aborting.expedition.target.too.far,Abortando Expedição. {0} de distancia.,,

```

Comment:
Mesma coisa aqui... `Aborting expedition. We are too far away from our target {0}`, e ainda coloca "distancia" sem acento. Desisto. Pode manter a sua.

Reply:
normal esquecer um acento ou outro , somos todos humanos aqui. eu vejo muito reporte do ingles pessoas nao entendendo o negocio de estar muito distante, é comum de so passar perto de um planeta ja abre a expedição mais o povo so vai terminar quando a nave ja esta muito distante do planeta, pq o destino é outro , aqui fica como teste , teria que ver como fica dentro do jogo, mais eu nunca tive esse problema , entao fica só na teoria.